index.xml

```diff
@@ -8661,15 +8786,15 @@ interface DedicatedWorkerGlobalScope : WorkerGlobalScope {
 
 <span class='comment'>// MathUtils is available for use in workers and on the main thread.</span>
 [Exposed=(Window,Worker)]
-interface MathUtils {
+namespace MathUtils {
   static double someComplicatedFunction(double x, double y);
 };
 
 <span class='comment'>// WorkerUtils is only available in workers.  Evaluating WorkerUtils
 // in the global scope of a worker would give you its interface object, while
```

Comment:
Talking about an "interface object" for a namespace seems a bit odd.

I think you should probably leave these as interfaces, or add interface examples in addition to namespace ones or something.